packages/web-extension/src/messaging/types.ts

```diff
@@ -134,3 +134,7 @@ export interface MessengerApiDependencies {
   messenger: Messenger;
   logger: Logger;
 }
+
+export interface ApiSourceChanger<T> {
```

Comment:
This is unused. Is there a missing test?

Reply:
It's garbage from the initial implementation. I will remove it. Thanks!

Reply:
removed in [d82fd39](https://github.com/input-output-hk/cardano-js-sdk/pull/489/commits/d82fd39a7586375a628367d7e2481fe5a631608d)